vllm/executor/ray_distributed_executor.py

```diff
@@ -65,6 +71,11 @@ class RayDistributedExecutor(DistributedExecutorBase):
 
     def _init_executor(self) -> None:
         self.forward_dag: Optional[ray.dag.CompiledDAG] = None
+
+        if (current_platform.is_xpu()
+                and "ONEAPI_DEVICE_SELECTOR" in os.environ):
+            del os.environ[current_platform.device_control_env_var]
```

Comment:
so when both `ZE_AFFINITY_MASK` and `ONEAPI_DEVICE_SELECTOR` envs exist, we respect `ONEAPI_DEVICE_SELECTOR` and why?